docs/tool_jmap.md

```diff
@@ -1,79 +1,77 @@
-﻿<!--
-* Copyright (c) 2017, 2022 IBM Corp. and others
-*
-* This program and the accompanying materials are made
-* available under the terms of the Eclipse Public License 2.0
-* which accompanies this distribution and is available at
-* https://www.eclipse.org/legal/epl-2.0/ or the Apache
-* License, Version 2.0 which accompanies this distribution and
-* is available at https://www.apache.org/licenses/LICENSE-2.0.
-*
-* This Source Code may also be made available under the
-* following Secondary Licenses when the conditions for such
-* availability set forth in the Eclipse Public License, v. 2.0
-* are satisfied: GNU General Public License, version 2 with
-* the GNU Classpath Exception [1] and GNU General Public
-* License, version 2 with the OpenJDK Assembly Exception [2].
-*
-* [1] https://www.gnu.org/software/classpath/license.html
-* [2] http://openjdk.java.net/legal/assembly-exception.html
-*
-* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
-* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
--->
-
-# Java memory map (`jmap`) tool
-
-Use the `jmap` tool to get memory information for a particular Java&trade; process, or list of processes. The tool shows statistics about classes on the heap, including the number of objects and their aggregate size. The command syntax is as follows:
-
-    jmap [<option>] [<vmid>]
-
-`<vmid>` is the Attach API virtual machine identifier for the Java process. This ID is typically the same as
-the operating system *process ID*, unless you specified the `-Dcom.ibm.tools.attach.id` system property when you started the process.

```

Comment:
How come this entire file is showing as a difference, and this point was removed?

Reply:
Nm, I see it wasn't removed, just re-formatted.